kt-extensions/src/test/java/com/mindera/skeletoid/kt/extensions/utils/StringUnitTest.kt

```diff
@@ -232,4 +232,16 @@ class StringUnitTest {
 
         assertFalse(email.isEmailValid())
     }
+
+    @Test
+    fun testNullIfEmptyNull() {
+        val text: String? = null
+        assertEquals(null, text.nullIfEmpty())
+    }
+
+    @Test
+    fun testNullIfEmptyEmpty() {
+        val text: String = ""
+        assertEquals(null, text.nullIfEmpty())
+    }
```

Comment:
should we add a test for when a string is neither null nor empty?

Reply:
done!